1.10/s2i/bin/assemble

```diff
@@ -3,12 +3,15 @@
 set -e
 
 echo "---> Installing application source"
-cp -Rf /tmp/src/. ./
+mv /tmp/src/* ./
```

Comment:
Can we leave the `cp` for now, until we choose what to do about this for all images?

Similarly to other versions